src/systemd/sd-messages.h

```diff
@@ -41,6 +41,7 @@ _SD_BEGIN_DECLARATIONS;
 
 #define SD_MESSAGE_COREDUMP         SD_ID128_MAKE(fc,2e,22,bc,6e,e6,47,b6,b9,07,29,ab,34,a2,50,b1)
 #define SD_MESSAGE_TRUNCATED_CORE   SD_ID128_MAKE(5a,ad,d8,e9,54,dc,4b,1a,8c,95,4d,63,fd,9e,11,37)
+#define SD_MESSAGE_EXCEPTION        SD_ID128_MAKE(1f,4e,0a,44,a8,86,49,93,9a,ae,a3,4f,c6,da,8c,95)
```

Comment:
hmm, you are calling this "exception" here... I think we should stick to a single terminology here. I.e. either call the message id "stacktrace" or call the command line switch "--exception" too.

I am mostly leaning towards" "stacktrace for both though.